src/payment/checkout/Checkout.jsx

```diff
@@ -156,12 +156,26 @@ class Checkout extends React.Component {
 
     // Stripe element config
     // TODO: Move these to a better home
-    const appearance = {
-      theme: 'stripe',
-    };
     const options = {
       clientSecret: this.props.clientSecretId,
-      appearance,
+      appearance: {
+        rules: {
+          '.Input': {
+            border: 'solid 1px #707070',
+            borderRadius: '0.375rem',
+          },
+          '.Input:hover': {
+            border: 'solid 1px #1f3226',
+          },
+          '.Input:focus': {
+            color: '#454545',
+            backgroundColor: '#FFFFFF',
+            borderColor: '#0A3055',
+            outline: '0',
+            boxShadow: '0 0 0 1px #0A3055',
+          },
+        },
+      },
```

Comment:
Usually on MFE's styling is added either as utility classes from Bootstrap or through a SCSS file, which is where we would add colors like `$gray-300`, which would get pulled from Paragon as opposed to hard coding the hex color. This is because in case our design system changes the hex value for a specific color, we wouldn't have to update it here - however, this is not always followed.

I think it's ok to have them here considering Stripe is very particular, just wanted to give some edX background context. For the hex colors, where did you get them? Specifically looking at `borderColor` and the `border`? I couldn't find them under Paragon.

Reply:
yeah for stripe from what I can tell the styling is to be passed in through the appearance object.  Is there a way to get those variables in the js? I tried to look to see if any of the variables were exported but it just looks like components are in paragon?

The values came from the fields above for the cardHolderInformation.  Set the stripe elements to match those.